Frontend/tests/test_messagemanager.cpp

```
@@ -29,8 +29,10 @@ TEST_CASE("Test MessageManager getChatMessages") {
   TestMessageManager message_manager(&network_manager, url, &entity_factory, timeout_ms);
 
   QJsonArray messages_array{
-      QJsonObject{{"id", 1}, {"sender_id", 10}, {"text", "Hello"}, {"timestamp", "2025-11-03T12:00:00Z"}},
-      QJsonObject{{"id", 2}, {"sender_id", 11}, {"text", "Hi"}, {"timestamp", "2025-11-03T12:01:00Z"}}};
+      QJsonObject{
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_SLOTS is a macro then please configure it.

(unknownMacro)